crates/mun_compiler/src/diagnostics/emit.rs

```diff
@@ -42,7 +42,7 @@ impl Emit for Diagnostic {
         writer.set_color(&error)?;
         write!(writer, "error")?;
 
-        // Write diagnostic message
+        // Write diagnostics message
```

Comment:
singular is diagnostic, plural is diagnostics.

Reply:
I think I renamed a field which also renamed comments ...